ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -1224,6 +1230,8 @@ export default class ConfirmTransactionBase extends Component {
           onCancelAll={() => this.handleCancelAll()}
           onCancel={() => this.handleCancel()}
           onSubmit={() => this.handleSubmit()}
+          onApprove={() => this.handleApprove()}
```

Comment:
Because this is not used for all token approvals, the code would be clearer if this was renamed to indicate that it is only used for `setApprovalForAll` (e.g. `onSetApprovalForAll` and `handleSetApprovalForAll`)

Reply:
Done